app-backend/app/src/main/scala/utils/Config.scala

```diff
@@ -1,24 +1,17 @@
 package com.azavea.rf.utils
 
 import com.typesafe.config.ConfigFactory
-
+import com.azavea.rf.database.Config
```

Comment:
I'm getting a compiler warning here -- is this import doing anything? It should be masked by the defined trait.